crates/ruff_linter/src/rules/flake8_pie/snapshots/ruff_linter__rules__flake8_pie__tests__preview__PIE800_PIE800.py.snap

```diff
@@ -0,0 +1,108 @@
+---
+source: crates/ruff_linter/src/rules/flake8_pie/mod.rs
+---
+PIE800.py:1:14: PIE800 [*] Unnecessary spread `**`
+  |
+1 | {"foo": 1, **{"bar": 1}}  # PIE800
+  |              ^^^^^^^^^^ PIE800
+2 | 
+3 | foo({**foo, **{"bar": True}})  # PIE800
+  |
+  = help: Remove unnecessary dict
+
+ℹ Safe fix
+1   |-{"foo": 1, **{"bar": 1}}  # PIE800
+  1 |+{"foo": 1, "bar": 1}  # PIE800
+2 2 | 
+3 3 | foo({**foo, **{"bar": True}})  # PIE800
+4 4 | 
+
+PIE800.py:3:15: PIE800 [*] Unnecessary spread `**`
+  |
+1 | {"foo": 1, **{"bar": 1}}  # PIE800
+2 | 
+3 | foo({**foo, **{"bar": True}})  # PIE800
+  |               ^^^^^^^^^^^^^ PIE800
+4 | 
+5 | {**foo, **{"bar": 10}}  # PIE800
+  |
+  = help: Remove unnecessary dict
+
+ℹ Safe fix
+1 1 | {"foo": 1, **{"bar": 1}}  # PIE800
+2 2 | 
+3   |-foo({**foo, **{"bar": True}})  # PIE800
+  3 |+foo({**foo, "bar": True})  # PIE800
+4 4 | 
+5 5 | {**foo, **{"bar": 10}}  # PIE800
+6 6 | 
+
+PIE800.py:5:11: PIE800 [*] Unnecessary spread `**`
+  |
+3 | foo({**foo, **{"bar": True}})  # PIE800
+4 | 
+5 | {**foo, **{"bar": 10}}  # PIE800
+  |           ^^^^^^^^^^^ PIE800
+6 | 
+7 | { # PIE800
+  |
+  = help: Remove unnecessary dict
+
+ℹ Safe fix
+2 2 | 
+3 3 | foo({**foo, **{"bar": True}})  # PIE800
+4 4 | 
+5   |-{**foo, **{"bar": 10}}  # PIE800
+  5 |+{**foo, "bar": 10}  # PIE800
+6 6 | 
+7 7 | { # PIE800
+8 8 |     **foo,
+
+PIE800.py:9:7: PIE800 [*] Unnecessary spread `**`
+   |
+ 7 |   { # PIE800
+ 8 |       **foo,
+ 9 |       **{
+   |  _______^
+10 | |         "bar": 10
+11 | |     },
+   | |_____^ PIE800
+12 |   }
+   |
+   = help: Remove unnecessary dict
+
+ℹ Safe fix
+6  6  | 
+7  7  | { # PIE800
+8  8  |     **foo,
+9     |-    **{
+   9  |+    
+10 10 |         "bar": 10
+11    |-    },
+   11 |+    ,
```

Comment:
Is there a requirement that the output be formatted nicely? The edit here is syntactically valid, but the formatting is quite off after the fix (not sure if that's a problem or if we could juts assume people will run the formatter afterwards).

Reply:
It's generally seen as "best effort" but not required.